### ISSUE TITLE
Add button to remove template code snippet

### ISSUE THREAD

**Issue by s-and-witch** [haskell/play-haskell#32]
### Feature request

Each time you enter the play.haskell.org site you can see the template code snippet like this:
```haskell
import Data.List (partition)

main :: IO ()
main = do
  let unsorted = [10,9..1]
  putStrLn $ show $ quicksort unsorted

quicksort :: Ord a => [a] -> [a]
quicksort []     = []
quicksort (x:xs) = let (lesser, greater) = partition (<= x) xs
                   in quicksort lesser ++ [x] ++ quicksort greater
```
Sometimes it's annoying to remove the code and type the "minimal" main by hand:
```
main :: IO ()
main = _
```
So, I'm asking for the button to remove standard template snippet and add the minimal one.

## Alternative
Just remove all the template snippets and add hello world as the new one for the :
```haskell
main :: IO ()
main = putStrLn "Hello, World!"
```

**Comment by tomsmeding** [comment]
@s-and-witch You're right, I've had this annoyance as well. I've hacked together a button (live at the usual https://play.haskell.org/); what do you think? It fades away 2 seconds after the user did something (mousemove / keydown) so as to not remain in the way.

**Comment by s-and-witch** [comment]
That looks fine for me.

**Comment by tomsmeding** [comment]
Thanks!
